pkg/cephmgr/rgw/leader.go

```diff
@@ -163,6 +163,24 @@ func markUnapplied(context *clusterd.Context) error {
 }
 
 func (r *Leader) enable(context *clusterd.Context, factory client.ConnectionFactory) error {
```

Comment:
enable() is still only called in the standalone/etcd world. The operator just has to create the rgw keyring and start the rgw service.